clip_win.cpp

```diff
@@ -416,22 +640,11 @@ size_t lock::impl::get_data_length(format f) const {
 
 #if CLIP_ENABLE_IMAGE
 
-bool lock::impl::set_image(const image& image) {
-  const image_spec& spec = image.spec();
-
-  // Add the PNG clipboard format for images with alpha channel
-  // (useful to communicate with some Windows programs that only use
-  // alpha data from PNG clipboard format)
-  if (spec.bits_per_pixel == 32 &&
-      spec.alpha_mask) {
-    UINT png_format = RegisterClipboardFormatA("PNG");
-    if (png_format) {
-      Hglobal png_handle(win::write_png(image));
-      if (png_handle)
-        SetClipboardData(png_format, png_handle);
-    }
-  }
+namespace win {
```

Comment:
I know we don't need this namespace here and we could move create_img to the other namespace block. I just did this because the code comparison is actually easier when this code is here (when using vscode to compare changes). If you don't see the point in having this, I will be happy to move it.